lib/threeGenerator.js

```diff
@@ -94,7 +94,7 @@ const get3dShape = (vertices, material, holes = []) => {
   referenceObject.quaternion.conjugate();
   referenceObject.updateMatrixWorld();
 
-  // What does this line do? If it is modifying inline then forEach() is preferable to map()
+  // What does this line do? If it is modifying inline then forEach() is preferable to map() << Theo: will check. looks like error
```

Comment:
Can we use a @todo tag here to keep track of what needs done rather than have a conversation in the code comment? 😄 Here's an [example of the implementation I've got in mind](http://usejsdoc.org/tags-todo.html).